src/mongocxx/test_util/client_helpers.cpp

```diff
@@ -483,6 +485,30 @@ bool should_run_client_side_encryption_test(void) {
         return false;
     }
 
+    auto email = getenv("MONGOCXX_TEST_GCP_EMAIL");
+    auto private_key = getenv("MONGOCXX_TEST_GCP_PRIVATEKEY");
+
+    if (!email || !private_key) {
+        FAIL(
+            "Please set environment variables for client side encryption tests:\n"
+            "\tMONGOCXX_TEST_GCP_EMAIL\n"
+            "\tMONGOCXX_TEST_GCP_PRIVATEKEY\n"
+            "\n");
```

Comment:
The current behavior is to skip CSFLE tests if any of the AWS environment variables are set.

The proposed new behavior is to fail if AWS environment variables are set, but Azure or GCP are not set.

I suggest tweaking this to `FAIL` if some, but not all, of the required CSFLE environment variables are set. And skip if none are set.